src/contracts.rs

```diff
@@ -382,20 +444,23 @@ where
 impl<T1, T2, T3, E1, E2, E3, C, T4, E4, E5, T6, E6, Q> Contract<C, Q>
     for ContractWrapper<T1, T2, T3, E1, E2, E3, C, Q, T4, E4, E5, T6, E6>
 where
-    T1: DeserializeOwned + Debug + Clone,
-    T2: DeserializeOwned + Debug + Clone,
-    T3: DeserializeOwned + Debug + Clone,
-    T4: DeserializeOwned,
-    T6: DeserializeOwned,
-    E1: Display + Debug + Send + Sync + Error + 'static,
-    E2: Display + Debug + Send + Sync + Error + 'static,
-    E3: Display + Debug + Send + Sync + Error + 'static,
-    E4: Display + Debug + Send + Sync + 'static,
-    E5: Display + Debug + Send + Sync + 'static,
-    E6: Display + Debug + Send + Sync + 'static,
-    C: CustomMsg,
-    Q: CustomQuery + DeserializeOwned,
+    T1: DeserializeOwned + 'static, // Type of message passed to `execute` entry-point.
```

Comment:
The less we are strict in the generics bounds the better. This additional `'static` in a trait's types is not necessary.

Reply:
Removed `'static` from trait type's bounds.